app/_locales/en/messages.json

```diff
@@ -434,9 +434,16 @@
   "beCareful": {
     "message": "Be careful"
   },
+  "betaHomeHeaderText": {
+    "message": "This is a BETA version. Please report bugs $1",
+    "description": "$1 represents the word 'here'"
```

Comment:
nit:
I think this can be rephrased. 

$1 represents an hyperlink  to bug website